packages/react-scripts/fixtures/kitchensink/integration/env.test.js

```diff
@@ -6,7 +6,15 @@ describe('Integration', () => {
     it('file env variables', async () => {
       const doc = await initDOM('file-env-variables')
 
-      expect(doc.getElementById('feature-file-env-variables').textContent).to.equal('fromtheenvfile.')
```

Comment:
Seems that dot makes no sens. 
btw. going to change env variables names and values at tests to make code more readable